test/services/storage/processor.spec.ts

```diff
@@ -290,7 +290,7 @@ describe('Storage services: Events Processor', () => {
         expect(agreement?.billingPrice).to.be.eql(plan.amount)
         expect(agreement?.availableFunds).to.be.eql(event.returnValues.availableFunds)
         expect(agreement?.lastPayout).to.be.eql(await getBlockDate(eth, event.blockNumber))
-        expect(agreementServiceEmitSpy).to.have.been.calledOnceWith('created')
+        expect(agreementServiceEmitSpy).to.have.been.calledWith('created')
```

Comment:
Interesting why did you change from `calledOnceWith` to `calledWith`? Shouldn't it be called only once?